part-1-ajax/test.js

```diff
@@ -1,6 +1,15 @@
 function getCountryName(code) {
     $.ajax({
       //uses the three letter code to get the country's full name
+      var rootUrl1 = "https://restcountries.eu/rest/v1/alpha/",
```

Comment:
Thank you Charles and Nate for the feedback. In truth, as much as I studied
as well as having Joshua running through it several times. I could not
figure out how to execute ajax & apis by the question wording or general.

I know I failed the test. I'll keep working on it.

Enjoy the weekend.

On Sep 9, 2016 12:36 PM, "Charles Kuang" <notifications@github.com> wrote:

In part-1-ajax/test.js
<https://github.com/C4Q/ac3.1-exam-4/pull/26#discussion_r78210297>:

> @@ -1,6 +1,15 @@
>  function getCountryName(code) {
>      $.ajax({
>        //uses the three letter code to get the country's full name
> +      var rootUrl1 = "https://restcountries.eu/rest/v1/alpha/",

you shouldn't be defining variables inside of the ajax options

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/C4Q/ac3.1-exam-4/pull/26/files/ff1c08d097c95432080037f652905c4535bd1c45#r78210297>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ATfCa1eYHwfZLsNY7XXkgcPZsz2Z4pC1ks5qoYsNgaJpZM4J4cxi>
.